packages/utils/src/instrument/fetch.ts

```diff
@@ -158,7 +175,8 @@ async function streamHandler(response: Response): Promise<void> {
     return;
   }
 
-  await resolveResponse(clonedResponseForResolving, () => {
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  resolveResponse(clonedResponseForResolving, () => {
```

Comment:
Why do we want leave this floating?

Reply:
It is fine to have floating promises if you are certain that they cannot reject - i think this is the case here!